router/throttler/config.go

```diff
@@ -0,0 +1,50 @@
+package throttler
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/rudderlabs/rudder-go-kit/config"
+)
+
+type normalConfig struct {
```

Comment:
```suggestion
type staticRateLimitConfig struct {
```
normalConfig is seems very generic

Reply:
or `staticThrottleConfig`

Reply:
Done